src/Persistence/Sql/Query.php

```diff
@@ -0,0 +1,339 @@
+<?php
+
+declare(strict_types=1);
+
+namespace atk4\data\Persistence\Sql;
+
+use atk4\data\Exception;
+use atk4\data\Field;
+use atk4\data\FieldSqlExpression;
+use atk4\data\Model;
+use atk4\data\Persistence;
+use atk4\dsql\Expression;
+use atk4\dsql\Expressionable;
+use atk4\dsql\Query as DsqlQuery;
+
+/**
+ * @method Query get()
```

Comment:
anotations not needed, use phpdoc above implemented methods directly

Reply:
Note `\atk4\data\Persistence\Query\Sql` does not extend `\atk4\dsql\Query`, it just forwards the calls to it

Reply:
but it impl. `QueryInterface`